changelog/fragments/3319-deprecate-migrate-subcommand.yaml

```diff
@@ -0,0 +1,9 @@
+entries:
+  - description: >
+      The migrate sub-command is deprecated
+    kind: deprecation
+    breaking: true
```

Comment:
Not sure if deprecation can be a breaking change. I think this should be `false`.